src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.StateSet.cs

```diff
@@ -24,166 +18,33 @@ private sealed class StateSet
             public readonly string Language;
             public readonly DiagnosticAnalyzer Analyzer;
 
-            private readonly ConcurrentDictionary<DocumentId, ActiveFileState> _activeFileStates;
+            private readonly ConcurrentSet<DocumentId> _activeDocuments;
```

Comment:
What do we even need active docs for here?

Reply:
I don't know. It's still used.